tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerApiParser.csproj

```diff
@@ -7,7 +7,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>swagger_api_parser</RootNamespace>
     <LangVersion>Latest</LangVersion>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.0.1</VersionPrefix>
```

Comment:
Could we add a changelog file to the folder? i.e https://github.com/Azure/azure-sdk-tools/blob/main/tools/apiview/emitters/cadl-apiview/CHANGELOG.md

Reply:
Good. Added changelog